data/src/test/assets/api_mocks/feedback_reporting.json

```diff
@@ -19,18 +31,17 @@
     "network_type": "wifi",
     "device_lang": "EN"
```

Comment:
This and the `phone_type` above can both be removed too

Reply:
Done.

Reply:
Resolving on behalf of @jcqli  as the change has been approved.